StripePayments/StripePayments/Source/API Bindings/Models/Shared/STPIntentActionMultibancoDisplayDetails.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  STPIntentActionMultibancoDisplayDetails.swift
+//  StripePayments
+//
+//  Created by Nick Porter on 4/22/24.
+//
+
+import Foundation
+
+/// Contains Multibanco details necessary for the customer to complete the payment.
+public class STPIntentActionMultibancoDisplayDetails: NSObject, STPAPIResponseDecodable {
+    /// The multibanco entity number
+    @objc public let entity: String
+
+    /// Multibanco reference number
+    @objc public let reference: String
+
+    /// The expiry date of the multibanco voucher.
+    @objc public let expiresAt: Date
+
+    /// The URL to the hosted multibanco voucher page, which allows customers to view the multibanco voucher.
+    @objc public let hostedVoucherURL: URL
+
+    /// :nodoc:
+    public private(set) var allResponseFields: [AnyHashable: Any]
+
+    /// :nodoc:
+    @objc public override var description: String {
+        let props: [String] = [
+            // Object
+            String(
+                format: "%@: %p",
+                NSStringFromClass(STPIntentActionMultibancoDisplayDetails.self),
+                self
+            ),
+            // MultibancoDisplayDetails
+            "number = \(String(describing: entity))",
```

Comment:
is calling the entity a 'number' intentional?

Reply:
Nope! Good catch.